Library/Homebrew/env_config.rb

```diff
@@ -295,6 +295,9 @@ module EnvConfig
                       "or `$HOME/.homebrew/livecheck_watchlist.txt` otherwise.",
         default:      "#{ENV.fetch("HOMEBREW_USER_CONFIG_HOME")}/livecheck_watchlist.txt",
       },
+      HOMEBREW_LOCK_CONTEXT:                     {
+        description: "If set, Homebrew will add this output as additional context for locking errors.",
```

Comment:
This, or similar, might help make the variable less cryptic:
```suggestion
        description: "If set, Homebrew will add this output as additional context for locking errors. This is useful when running `brew` in the background.",
```